fortress/ros2_integration.md

```diff
@@ -16,12 +18,12 @@ required to go further.
 
 ## Bidirectional communication
 
-We can initialize a bidirectional bridge so we can have ROS as the publisher and Ignition as the subscriber or vice versa.
+We can initialize a bidirectional bridge so we can have ROS as the publisher and Ignition as the subscriber or vice versa. The syntax is `/TOPIC@ROS_MSG@IGN_MSG`, such that `TOPIC` is the Ignition internal topic, `ROS_MSG` is the ROS message type for this topic, and `IGN_MSG` is the Ignition message type.
 
 For example:
 
 ```
-ros2 run ros_ign_bridge parameter_bridge /TOPIC@ROS_MSG@IGN_MSG
+ros2 run ros_ign_bridge parameter_bridge /scan@sensor_msgs/msg/LaserScan@ignition.msgs.LaserScan
```

Comment:
I think its important to show an actual use case when introducing topics like this. Otherwise, its not clear that `ROS_MSG` means `sensor_msgs/msg/LaserScan` and `IGN_MSG` means `ignition.msgs.LaserScan` (notable because one wants slashes and the other wants dots).